GenderPayGap.WebUI/ErrorHandling/CustomErrorPageExceptions.cs

```diff
@@ -24,6 +24,14 @@ public class EmailNotVerifiedException : CustomErrorPageException
         public DateTime? EmailVerifySendDate { get; set; }
     }
 
+    public class UserNotRegisteredToReportForOrganisationException : CustomErrorPageException
+    {
+        public override string ViewName => "";

```

Comment:
This needs your new view name (`../Errors/UserNotRegisteredToReportForOrganisation`)